pycram/src/pycram/robot_descriptions/hsrb_states.py

```diff
@@ -0,0 +1,73 @@
+from semantic_digital_twin.datastructures.prefixed_name import PrefixedName
+from semantic_digital_twin.robots.hsrb import HSRB
+
+from ..datastructures.enums import (
+    StaticJointState,
+    Arms,
+    GripperState as GripperStateEnum,
+    TorsoState,
+)
+from ..joint_state import JointState, ArmState, GripperState, JointStateManager
+
+left_arm = ArmState(
+    name=PrefixedName("hsrb", "arm"),
+    joint_names=[
+        "arm_flex_joint",
+        "arm_roll_joint",
+        "wrist_flex_joint",
+        "wrist_roll_joint"
+    ],
+    joint_positions=[0.0, 1.5, -1.85, 0.0],
+    state_type=StaticJointState.Park,
+    arm=Arms.LEFT,
+)
+
+left_gripper_open = GripperState(
+    name=PrefixedName("hsrb", "left_gripper_open"),
+    joint_names=["hand_l_proximal_joint", "hand_r_proximal_joint", "hand_motor_joint"],
+    joint_positions=[0.3, 0.3, 0.3],
```

Comment:
@Tigul how does pycram work interally? cause this should be mimic joints if i'm not mistaken, meaning you only need to set one of them, prob hand_motor_joint.

Reply:
But the whole robot states thing is only an intermediate solution until I merge this into the AbstractRobot annotations